<launch>

	<arg name="duration" default="25"/>
	<arg name="rec_name" default="SR_detection.bag"/>
	<arg name="package_name" default="survey_and_rescue"/>
	<arg name="path_rel_to_pkg" default="scripts"/>
	<arg name="start_on_base" default="False"/>
	<arg name="start_with_countdown" default="True"/>
	<arg name="countdown" default="2"/>
	<arg name="stats_topic_rate" default="15"/>
	<arg name="num_beacons" default="8"/>
	<arg name="print_time" default="True"/>
	<arg name="record" default="false"/>

	<!-- <node name="edroneclient" type="edroneclient" pkg="edrone_client"/>
	<node name="drone_control" type="position_hold.py" pkg="survey_and_rescue" output="screen" />
	 -->
	<!-- <node pkg="survey_and_rescue" type="demo_scheduler.py" name="sr_scheduler"  output="screen"/> -->
  <node pkg="survey_and_rescue" type="beacon_detector.py" name="sr_beacon_detector" output="screen"/>
	
	<node pkg="survey_and_rescue" type="monitor_org_config.pyc" name="monitor" output="screen">
		<param name="~start_on_base" value="$(arg start_on_base)"/>
		<param name="~start_with_countdown" value="$(arg start_with_countdown)"/>
		<param name="~countdown" value="$(arg countdown)"/>
		<param name="~package_name" value="$(arg package_name)"/>
		<param name="~path_rel_to_pkg" value="$(arg path_rel_to_pkg)"/>	
		<param name="~stats_topic_rate" value="$(arg stats_topic_rate)"/>
		<param name="~print_time" value="$(arg print_time)"/>
		<param name="~num_beacons" value="$(arg num_beacons)"/>
	</node>
	
</launch>